test/test_network_constructors.jl

```diff
@@ -103,6 +104,8 @@ end
         psi_constraint_test(ps_model, constraint_names)
         psi_checkobjfun_test(ps_model, objfuncs[ix])
         psi_checksolve_test(ps_model, [MOI.ALMOST_OPTIMAL, MOI.OPTIMAL])
+        # TODO: add values from Bθ formulation to compare
```

Comment:
Please add this test before merging

Reply:
I added that test but just forgot to remove that TODO